tests/test_my_funcs.py

```diff
@@ -7,13 +8,13 @@ def test_print_hello_world():
 
 
 def test_my_func():
-    mf.my_func(True)
-    mf.my_func(False)
+    my_func(True)
+    my_func(False)
 
 
 def test_add_numbers():
-    assert mf.add_numbers(1, 2) == 3
+    assert add_numbers(1, 2) == 3
     with pytest.raises(TypeError):
-        mf.add_numbers("1", 3)
+        add_numbers("1", 3)
     with pytest.raises(TypeError):
-        mf.add_numbers(1, "3")
+        add_numbers(1, "3")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['add_numbers' may be undefined, or defined from star imports: mypackage.my_funcs](https://app.codacy.com/app/volker.hirthammer/pythonTest/pullRequest?prid=4697838)